docs/reference/commandline/cli.md

```diff
@@ -265,6 +280,23 @@ Following is a sample `config.json` file:
 {% endraw %}
 ```
 
+### Experimental features
+
+To enable experimental features, edit the `config.json` file and set
+`experimental` to `enabled`. The example below enables experimental features
+in a `config.json` file that already enables a debug feature.
+
+```json
+{
+  "experimental": "enabled",
+  "debug": true
+}
+```
+
+You can also enable experimental features from the Docker Desktop menu. See the
+[Docker Desktop Getting Started page](https://github.com/docker/docker.github.io/blob/master/docker-for-mac/index.md#experimental-features)
```

Comment:
We should probably link to the docs.docker.com site instead (otherwise, once this is published on docs.docker.com, we direct people to GitHub instead of staying on docs.docker.com);
```suggestion
[Docker Desktop Getting Started page](https://docs.docker.com/docker-for-mac/#experimental-features)
```